src/components/Input/Input.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { InputHTMLAttributes, FC } from 'react';
+import { classNames } from '../../utils/classNames';
+import Icon from '../Icon/Icon';
+
+interface IInputProps {
+  /** set customized style */
+  className?: string;
+  /** set input bar size */
+  inputSize?: 'mini' | 'small' | 'large' | 'big' | 'huge' | 'massive';
+  /** make input bar always focused */
+  focus?: boolean;
+  /** disabled the input bar */
+  disabled?: boolean;
+  /** apply error style to input bar */
+  error?: boolean;
+  /** make input bar transparent */
+  transparent?: boolean;
+  /** make input bar take whole space of parent container */
+  fluid?: boolean;
+  /** add specific icon to input bar */
+  icon?: string | { [key: string]: string | boolean };
+  /** apply loading style to input bar */
+  loading?: boolean;
+  /** set icon to show on left or right, default is right */
+  iconPosition?: 'left' | 'right';
+}
+
+export type PatInputProps = IInputProps & InputHTMLAttributes<HTMLInputElement>;
+
+const Input: FC<PatInputProps> = (props) => {
```

Comment:
Add a description about how to import it.

Reply:
added